src/rules/objectLiteralSortKeysRule.ts

```diff
@@ -184,6 +193,10 @@ function walk(ctx: Lint.WalkContext<Options>, checker?: ts.TypeChecker): void {
     }
 }
 
+function hasNewLineBefore(sourceFile: ts.SourceFile, element: ts.ObjectLiteralElement) {
+    return /\r?\n\r?\n/.test(sourceFile.text.slice(element.getFullStart(), element.getStart(sourceFile)));
```

Comment:
This will also match for blank lines inside comments, for example:

```ts
let foo = {
    foo: 1,
    /* I need some space to express myself

       and here the comment continues */
    bar: 2,
};
```
You can have a look at `newline-before-return` to see how I would check for a blank line before, between or after comments (but not inside).
https://github.com/palantir/tslint/blob/master/src/rules/newlineBeforeReturnRule.ts#L62-L81